vignettes/EnrichmentBrowser.Rmd

```diff
@@ -0,0 +1,989 @@
+expression values as the average of the corresponding probe values.
+
+```{r probe2gene}
+allSE <- probe2gene(all.eset)
+head(rownames(allSE))
+```
+
+Note, that the mapping from probe to gene is done automatically as long
+as as you have the corresponding annotation package, here the `r Biocpkg("hgu95av2.db")` package, installed. Otherwise, the mapping can be manually defined in the `rowData` slot.
+
+```{r show-probe2gene}
+ rowData(se)
+```
+
+## RNA-seq data
+
+To demonstrate the functionality of the package for RNA-seq data, we
+consider transcriptome profiles of four primary human airway smooth
+muscle cell lines in two conditions: control and treatment with
+dexamethasone [@Himes14].
+
+We load the `r Biocpkg("airway")` dataset
+
+```{r load-airway}
+library(airway) 
+data(airway)
+```
+
+For further analysis, we remove genes with very low read counts and
+measurements that are not mapped to an ENSEMBL gene ID.
+
+```{r preproc-airway}
+airSE <- airway[grep("^ENSG", rownames(airway)),] 
+airSE <- airSE[rowSums(assay(airSE)) > 4,]
+dim(airSE) 
+assay(airSE)[1:4,1:4]
+```
+
+# Normalization
+
+Normalization of high-throughput expression data is essential to make
+results within and between experiments comparable. Microarray (intensity
+measurements) and RNA-seq (read counts) data typically show distinct
+features that need to be normalized for. The function `normalize` wraps commonly
+used functionality from  `r Biocpkg("limma")` for microarray normalization and from `r Biocpkg("EDASeq")` for RNA-seq normalization. For specific needs that deviate from these standard normalizations, the user should always refer to more specific functions/packages.
+
+Microarray data is expected to be single-channel. For two-color arrays,
+it is expected that normalization within arrays has been already carried
+out, e.g. using from `normalizeWithinArrays` from `r Biocpkg("limma")`.
+
+A default quantile normalization based on `normalizeBetweenArrays` from `r Biocpkg("limma")` can be carried out via
+
+```{r norm-ma}
+ allSE <- normalize(allSE, norm.method = "quantile")
+```
+
+
+```{r plot-norm, fig.width=12, fig.height=6}
+par(mfrow=c(1,2))
+boxplot(assay(allSE, "raw")) 
+boxplot(assay(allSE, "norm"))
+```
+
+Note that this is only done for demonstration, as the ALL data has been
+already RMA-normalized by the authors of the ALL dataset.
+
+RNA-seq data is expected to be raw read counts. Note that normalization
+for downstream DE analysis, e.g.with `r Biocpkg("edgeR")` and `r Biocpkg("DESeq2")`, is not ultimately necessary
+(and in some cases even discouraged) as many of these tools implement
+specific normalization approaches themselves. See the vignette of `r Biocpkg("EDASeq")`, `r Biocpkg("edgeR")`,
+and `r Biocpkg("DESeq2")` for details.
+
+In case normalization is desired, between-lane normalization to adjust
+for sequencing depth can be carried out as demonstrated for microarray
+data.
+
+```{r norm-rseq}
```

Comment:
`eval = FALSE` should likely be added since it shouldn't display the output.

Reply:
okay